content/en/cosign/verify.md

```diff
@@ -210,6 +210,27 @@ MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEgrKKtyws86/APoULh/zXk4LONqII
 AcxvLtLEgRjRI4TKnMAXtIGp8K4X4CTWPEXMqSYZZUa2I1YvHyLLY2bEzA==
 -----END PUBLIC KEY-----
 ```
+## Custom Components
+
+For configuring Cosign to work with custom components, checkout the [Configuring Cosign with Custom Components](https://docs.sigstore.dev/cosign/custom_components/) docs to find out how to achieve this.
+
+### Custom Root Cert
+
+You can override the public good instance CA using the environment variable `SIGSTORE_ROOT_FILE`, e.g.
+
+```
+export SIGSTORE_ROOT_FILE="/home/jdoe/myrootCA.pem"
+```
+
+### SCM Integration
```

Comment:
Let's move SCM Integration and Attestations back to the overview, as I think they're better as high level concepts in the intro

Reply:
moved back.